isis/tests/FunctionalTestsLrowaccal.cpp

```diff
@@ -28,15 +28,28 @@ TEST(Lrowaccal, FunctionalTestLrowaccalRadianceUnitsLabelExists) {
     lrowaccal(options);
   }
   catch(IException &e) {
-    FAIL() << "Unable to open image cube: " << e.what() << std::endl;
+    FAIL() << "Call to lrowaccal failed, unable to calibrate cube: " << e.what() << std::endl;
   }
 
   Cube outCube(outCubeFileName);
   Pvl *outCubeLabel = outCube.label();
 
-  PvlGroup &radiometryGroup = outCubeLabel->findGroup("Radiometry", Pvl::Traverse);
+  PvlGroup radiometryGroup;
+  try {
+    radiometryGroup = outCubeLabel->findGroup("Radiometry", Pvl::Traverse);
+  }
+  catch(IException &e) {
+    FAIL() << "Unable to find Radiometry group in output cube label: " << e.what() << std::endl;
+  }
 
-  EXPECT_EQ(radiometryGroup["RadiometricType"].unit().toStdString(), "W/m2/sr/um");
+  PvlKeyword radiometricTypeKeyword;
+  try {
+    radiometricTypeKeyword = radiometryGroup.findKeyword("RadiometricType");
+  }
+  catch(IException &e) {
+    FAIL() << "Unable to find RadiometricType keyword in Radiometry group of output cube label: " << e.what() << std::endl;
+  }
```

Comment:
You can just do

```
ASSERT_TRUE(outCube.hasGroup("Radiometry"))
PvlGroup &radiometry = outCube.group("Radiometry");
ASSERT_TRUE(radiometry.hasKeyword("RadiometricType"))
```

Reply:
I see. Thank you. It is much more concise now.